azure_monitor/python_logger_opencensus_azure/monitoring/src/logger.py

```diff
@@ -104,6 +116,21 @@ def get_logger(self, component_name="AppLogger", custom_dimensions={}):
         handler = self._initialize_azure_log_handler(component_name, custom_dimensions)
         return self._initialize_logger(handler, component_name)
 
+    def get_event_logger(self, component_name="AppLogger", custom_dimensions={}):
+        """Get Event ogger Object.
+
+        Args:
+            component_name (str, optional): Name of logger. Defaults to "AppLogger".
+            custom_dimensions (dict, optional): {"key":"value"} to capture with every log.
+                Defaults to {}.
+
+        Returns:
+            Logger: A logger.
+        """
+        self.update_config(self.config)
+        handler = self._initialize_azure_event_handler(component_name, custom_dimensions)
+        return self._initialize_logger(handler, component_name)
```

Comment:
Should refactor this because _initialize_logger only uses HANDLER_NAME not EVENT_HANDLER_NAME

Reply:
This is addressed.